app/webpacker/components/Tickets/TicketComments.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { Button, Comment, Header } from 'semantic-ui-react';
+import { useQuery } from '@tanstack/react-query';
+import { DateTime } from 'luxon';
+import TicketCommentCreate from './TicketCommentCreate';
+import getComments from './api/getComments';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import '../../stylesheets/semantic/components/comment.min.css';
+
+export default function TicketComments({ ticketId, currentStakeholder }) {
+  const [createComment, setCreateComment] = useState(false);
+
+  const {
+    data: comments, isFetching, isError, refetch,
+  } = useQuery({
+    queryKey: ['ticket-comments', ticketId],
+    queryFn: () => getComments({ ticketId }),
```

Comment:
This "plain" fetching is dangerous, now that you've added pagination: Whenever there are more than 20 (or whatever is the page limit) comments, you would just happily display these 20 without realizing that there's more.

Either get rid of the pagination (for now) or implement it properly, please :)

Reply:
Okay, removed pagination for now.